packages/lib-classifier/src/store/WorkflowStepStore.js

```diff
@@ -151,8 +153,15 @@ const WorkflowStepStore = types
     }
 
     function setSteps (workflow) {
-      workflow.steps.forEach(([ stepKey, step ]) => {
-        self.steps.put(Object.assign({}, step, { stepKey }))
+      const { steps } = workflow
+      steps.forEach(([ stepKey, step ], index) => {
+        let next
+        if (steps[index + 1]) {
+          const nextStepKey = steps[index + 1][0]
```

Comment:
This is hard to follow. Why isn't the next step key `steps[index + 1]['stepKey']`?

Reply:
Maybe `const [ nextStepKey ] = steps[index + 1]` would be easier to read, if `steps` is an array of `[ stepKey, step]` pairs?

Reply:
Steps aren't stored in a map in Panoptes DB because JSON doesn't support it, thus at this point in the code it's `[[stepKey, step]]`. ADR 10 is about this.